deployment/keystone/depreciated.go

```diff
@@ -0,0 +1,127 @@
+package keystone
```

Comment:
I think you misnamed the file :) depreciated -> deprecated

Reply:
thx.